src/Bicep.Core.IntegrationTests/Emit/ParamsFileWriterTests.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Emit;
+using Bicep.Core.UnitTests.Assertions;
+using Bicep.Core.UnitTests.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+
+namespace Bicep.Core.IntegrationTests.Emit
+{
+    [TestClass]
+    public class ParamsFileWriterTests
+    {
+        
+        [DataTestMethod]
+        [DataRow(@"
```

Comment:
Do I need to also cover scenarios for "hello${123}world"? Also I am ignoring concatenation for the time being (since we are assuming that for initial version of the emitter, we won't have compile time constant folding)

Reply:
Ignoring interpolated strings for now is the right approach. We can come back and add that in when support for other expressions is added